web/src/components/navbar/NavBar.tsx

```diff
@@ -36,25 +41,38 @@ export const NavBar = (props: Props): ReactElement => {
     };
   }, []);
 
-  if (props.logoOnly) {
-    return <NavBarLogoOnly logoType={props.logoOnly} />;
-  } else if (props.landingPage && isLargeScreen) {
-    return <NavBarLandingDesktop />;
-  } else if (isLargeScreen) {
-    return <NavBarDesktop previousBusinessId={props.previousBusinessId} />;
-  } else {
-    return (
-      <>
+  const currentlyOnboarding = (): boolean => {
+    if (!router) {
+      return false;
+    }
+    return router.pathname === ROUTES.onboarding;
```

Comment:
I belive you're asking me to reverse the order of my statements?

What I have works in app because the so there is no precedence issue, I'm down to change it for style reasons though.